README.md

```diff
@@ -13,16 +7,107 @@ which has separate instructions.
 - Installs Drupal in a `web` subdirectory
 - Utilizes [composer-installers](https://github.com/composer/installers) and [Drupal Composer Scaffold](https://github.com/drupal/core-composer-scaffold) plugins
 - Can be used as project template for composer [create-project command](https://getcomposer.org/doc/03-cli.md#create-project):
-  ```
-  composer create-project az-digital/az-quickstart-scaffolding:2.5.x-dev azqs --no-interaction --no-dev
+
+## Notes
+If you are planning on pushing this site to Pantheon, you should use the
+[Pantheon upstream repository](https://github.com/az-digital/az-quickstart-pantheon) as your scaffolding repo,
+which has separate instructions.
+
+## Choosing the Right Branch for Your Quickstart Project
+In the Arizona Quickstart Scaffolding project, different branches serve various
+purposes. Here's a guide to understanding which branch you should choose:
+
+### Main Branch (`dev-main`)
+
+- **Purpose**: The main branch contains all the latest features and changes. It
+  represents the cutting-edge development and may be less stable than specific
+  release branches. You can use it with `dev-main` or an alias for the latest
+  unreleased version (e.g., `2.8.x`).
+- **When to Use**: Choose this branch if you want access to the very latest
+  features, and you are willing to work with potentially less-tested code. Ideal
+  for development, experimentation, or testing new functionalities.
```

Comment:
Alternate 
```suggestion
- **Purpose**: The main branch downloads the latest supported version of `az-digital/az_quickstart` and contains unreleased updates to the scaffolding repository. While it provides access to the latest stable features of Quickstart, it also includes cutting-edge changes to the scaffolding that may be less stable.
- **When to Use**: Choose this branch if you need a stable version of Quickstart for development or testing, and you want to work with the latest updates and improvements to the scaffolding itself.
```

Reply:
Should probably also mention that it only downloads the latest stable version at the time, and until the next minor release comes out.